pom.xml

```diff
@@ -255,6 +255,12 @@
             <version>${lombok.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.3.2</version>
```

Comment:
Add a property?

Reply:
Add a core level dependency for?

Reply:
jdk11 has removed it from rt.jar, that's he add this dependency here.

Reply:
> Add a core level dependency for?

Just to make it compile in JDK 9+, the scope is `provided` so nothing changed actually